src/command.c

```diff
@@ -207,6 +209,13 @@ handle_command_stop (const struct subcommand *sub,
 	g_print ("stopped container %s\n", config->optarg_container_id);
 
 out:
+	cgroup_dir = g_strdup_printf ("/sys/fs/cgroup/memory/%s",
```

Comment:
There are two occurrences of `"/sys/fs/cgroup/memory/%s"` so might be worth a macro for the static bit (or all of it), or even an macro / inline function for this.